tests/test_views.py

```diff
@@ -28,406 +28,444 @@
 def test_get_workflows(app, default_user):
     """Test get_workflows view."""
     with app.test_client() as client:
-        with patch(
-            "reana_server.rest.workflows.current_rwc_api_client",
-            make_mock_api_client("reana-workflow-controller")(),
-        ):
-            res = client.get(
-                url_for("workflows.get_workflows"),
-                query_string={"user_id": default_user.id_,
-                              "type": "batch"},
-            )
-            assert res.status_code == 403
-
-            res = client.get(
-                url_for("workflows.get_workflows"),
-                query_string={"access_token": default_user.access_token,
-                              "type": "batch"},
-            )
-            assert res.status_code == 200
+        mock_current_user = Mock(is_authenticated=False)
+        with patch('flask_login.utils._get_user',
```

Comment:
You can use the `mock.patch.multiple ` so it is a bit less indented :) see in [RWC](https://github.com/reanahub/reana-workflow-controller/blob/44b6d5eb9ed6e915f6e2294710eb6ef8c6fdd50a/tests/test_views.py#L1063-L1066).